src/KafkaFlow/Consumers/ConsumerWorker.cs

```diff
@@ -35,11 +36,8 @@ internal class ConsumerWorker : IConsumerWorker
             this.logHandler = logHandler;
             this.messagesBuffer = Channel.CreateBounded<IMessageContext>(consumer.Configuration.BufferSize);
 
-            this.workerStoppingEvent = new(logHandler);
-            this.workerStoppedEvent = new(logHandler);
-            this.workerProcessingEnded = new Event<IMessageContext>(logHandler);
-
             var middlewareContext = this.workerDependencyResolverScope.Resolver.Resolve<ConsumerMiddlewareContext>();
+            this.eventHub = this.workerDependencyResolverScope.Resolver.Resolve<IEventHub>() as EventHub;
```

Comment:
`this.workerDependencyResolverScope.Resolver.Resolve<EventHub>()`

Reply:
Done, please review